ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/install-on-bastionhost/provision/cucushift-installer-rehearse-gcp-ipi-install-on-bastionhost-provision-chain.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/install-on-bastionhost/provision/cucushift-installer-rehearse-gcp-ipi-install-on-bastionhost-provision-chain.yaml",
```

Comment:
I suggest we can place this provision chain under `gcp/ipi/cco-manual-workload-identity`. And `install-on-bastionhost` is not accurate enough, maybe we can rename it to `auth-with-sa`.